test/test-lib/instance.ts

```diff
@@ -246,6 +248,16 @@ export class DevProtocolInstance {
 		)
 	}
 
+	public async generateProperty(): Promise<void> {
+		const [proxfied] = await deployProxy(contract('Property'), this._deployer)
+		this._property = proxfied
+		await this.addressRegistry.setRegistry(
+			'Property',
+			this._property.address,
+			this.fromDeployer
+		)
```

Comment:
Why not move these processes inside "generatePropertyFactory"? Thus, "generateProperty" is no longer needed.